File: server/router/campus/campus_admin_staff.go
package campus

import (
	"github.com/flipped-aurora/gin-vue-admin/server/middleware"
	"github.com/gin-gonic/gin"
)

type CampusAdminStaffRouter struct{}

func (r *CampusAdminStaffRouter) InitCampusAdminStaffRouter(Router *gin.RouterGroup) {
	adminStaffRouter := Router.Group("campusAdminStaff").Use(middleware.OperationRecord())
	adminStaffRouterWithoutRecord := Router.Group("campusAdminStaff")
	{
		adminStaffRouter.POST("updateCampusAdminStaffStatus", campusAdminStaffApi.UpdateCampusAdminStaffStatus)
	}
	{
		adminStaffRouterWithoutRecord.GET("getCampusAdminStaffList", campusAdminStaffApi.GetCampusAdminStaffList)
		adminStaffRouterWithoutRecord.GET("findCampusAdminStaff", campusAdminStaffApi.FindCampusAdminStaff)
	}
}

File: web/src/api/campusAdminStaff.js
import service from '@/utils/request'

export const getCampusAdminStaffList = (params) => {
  return service({
    url: '/campusAdminStaff/getCampusAdminStaffList',
    method: 'get',
    params
  })
}

export const findCampusAdminStaff = (params) => {
  return service({
    url: '/campusAdminStaff/findCampusAdminStaff',
    method: 'get',
    params
  })
}

export const updateCampusAdminStaffStatus = (data) => {
  return service({
    url: '/campusAdminStaff/updateCampusAdminStaffStatus',
    method: 'post',
    data
  })
}
